portal/static/js/src/accountCreation.js

```diff
@@ -492,16 +492,26 @@ import ProcApp from "./modules/Procedures.js";
             });
             return arrCommunication;
         };
+        this.handleCurrentUser = function() {
+            let self = this;
+            CurrentUserObj.initCurrentUser(function() {
+                self.handleMedidataRave(CurrentUserObj.topLevelOrgs, CurrentUserObj.isAdminUser());
+            });
+        };
         this.getCurrentUserTopLevelOrgs = function() {
             var topLevelOrgs = OT.getUserTopLevelParentOrgs(leafOrgs);
             return topLevelOrgs.map(function(item) {
                 return OT.getOrgName(item);
             });
         };
-        this.handleMedidataRave = function() {
+        this.handleMedidataRave = function(orgs, noDisable) {
```

Comment:
Could you add a comment (if there isn't one already) describing why we're disabling editing?

Reply:
thanks, ivan, added